.github/workflows/run_torchbench.yml

```diff
@@ -0,0 +1,116 @@
+name: TorchBench CI
+on:
+  push:
+    tags:
+      - ciflow/torchbench/*
+  pull_request:
+    types: [ labeled ]
+  workflow_dispatch:
+
+env:
+  PYTHON_VERSION: "3.9"
+  # must be consistent with https://github.com/pytorch/benchmark/blob/main/requirements.txt#L19
+  NUMPY_VERSION: "1.23.1"
+  PR_NUM: ${{ github.event.number }}
+  PR_BODY: ${{ github.event.pull_request.body }}
+  PR_BASE_SHA: ${{ github.event.pull_request.base.sha }}
+  PR_HEAD_SHA: ${{ github.event.pull_request.head.sha }}
+  AWS_ACCESS_KEY_ID: ${{ secrets.AWS_OSSCI_METRICS_V2_ACCESS_KEY_ID }}
+  AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_OSSCI_METRICS_V2_SECRET_ACCESS_KEY }}
+
+jobs:
+  run-torchbench:
+    runs-on: [macos-torchbench]
+    # Set to 12 hours
+    timeout-minutes: 720
```

Comment:
When does this run get triggered. Also I don't imagine it taking longer than 8 hours. So if the run fails/hangs, we can get notified by morning..

Reply:
@kulinseth this gets triggered based on the following:
```
on:
  push:
    tags:
      - ciflow/torchbench/*
  pull_request:
    types: [ labeled ]
  workflow_dispatch:
```
- If you're tagging a PR with ciflow/torchbench it will start running the torchbench suite. 
- If you're pusing to a branch already tagged with ciflow/torchbench it will start running the torchbench suite
- Or triggering manually this workflow based on the workflow_dispatch

Reply:
It should take around 2 hours for a M2 pro with 32GB